fury/lib.py

```diff
@@ -21,6 +21,12 @@
 if have_jupyter_rfb:
     from rendercanvas.jupyter import RenderCanvas as JupyterWgpuCanvas
 
+IPython, have_ipython, _ = optional_package("IPython")
+if have_ipython:
+    from IPython.display import display as display_jupyter_widget
+else:
+    display_jupyter_widget = IPython
```

Comment:
This will cause runtime error if IPython (Optional dependency) is not installed, please handle the error here.

maybe implement a shell: 
```python
else:
    def display_jupyter_widget(*args, **kwargs):
        pass
```
and handle the error here it's used.

Reply:
It is exactly what we want, as it will create a Tripwire Error Due to The line no 24. 

Also if someone is trying to use jupyter, IPython is installed by default.